Network/Connection.cs

```diff
@@ -600,6 +605,30 @@ internal void Send(Packet packet, object instance, bool ignoreWriteLock)
             dataAvailableEvent.Set();
         }
 
+        /// <summary>
+        /// Sends the given <see cref="RawData"/> packet to the network.
+        /// </summary>
+        /// <param name="rawData">The <see cref="RawData"/> packet to send to the network.</param>
+        public void SendRawData(RawData rawData) => Send(rawData);
```

Comment:
Why is this method in the Connection.cs?

Reply:
makes more sense to put the SendXXX methods together? They all use the same underlying methods so they should (in my mind) be grouped together.

Reply:
The "sendraw" is an addon; thus belong to the addon partial class.

Reply:
seems like a core part of the library. i wouldnt really call it an addon. but idk

Reply:
i find it easier to reason about the behaviour of a method when its close to any methods it calls
so i placed them together

Reply:
I wouldn't call it a major feature; more a helper function to setup a quick and dirty communication. However, it isn't a part of the connection, since it relies on a specific packet implementation. (RawDataPacket) Furthermore, the method isn't called "send", but "sendraw". All in all it doesn't belong into the Connection.cs

Reply:
sound logic, good reasoning
will move it

Reply:
moved, check now